crates/holochain/src/core/ribosome/real_ribosome.rs

```diff
@@ -64,12 +64,18 @@ use crate::core::ribosome::host_fn::x_25519_x_salsa20_poly1305_encrypt::x_25519_
 use crate::core::ribosome::host_fn::x_salsa20_poly1305_decrypt::x_salsa20_poly1305_decrypt;
 use crate::core::ribosome::host_fn::x_salsa20_poly1305_encrypt::x_salsa20_poly1305_encrypt;
 use crate::core::ribosome::host_fn::zome_info::zome_info;
+use crate::core::ribosome::real_ribosome::wasmparser::Operator;
```

Comment:
for the sake of explicitness

```suggestion
use crate::core::ribosome::real_ribosome::wasmparser::Operator as WasmOperator;
```